test/arguments.js

```diff
@@ -1,3 +1,4 @@
+import {URL} from 'whatwg-url';
```

Comment:
It might be more complete to use [universal-url](https://npmjs.com/universal-url) here so that the Node v7 `URL` is included in tests when available.